package-lock.json

```diff
@@ -2531,6 +2536,11 @@
         "map-obj": "^1.0.0"
       }
     },
+    "can-use-dom": {
+      "version": "0.1.0",
```

Comment:
Please add package-lock json to gitignore so that it would not get updated all the times and hence would not affect every PR. Please correct this.

Reply:
Noted!